build.gradle

```diff
@@ -8,6 +8,59 @@ repositories {
 dependencies {
 }
 
+class ExecWait extends DefaultTask {
+	String command
+	String ready
+	String directory
+
+	@TaskAction
+	def spawnProcess() {
+
+		ProcessBuilder builder = new ProcessBuilder(command.split(' '))
+		builder.redirectErrorStream(true)
+		builder.directory(new File(directory))
+		Process process = builder.start()
+
+		InputStream stdout = process.getInputStream()
+		BufferedReader reader = new BufferedReader(new InputStreamReader(stdout))
+	}
+}
+
+task startDropServer(type: ExecWait) {
+	command './drop_server.py &'
+	ready 'Drop server running.'
+	directory './qabel-drop/'
+}
+
+task getNodeModules(type: ExecWait) {
+	command 'npm install'
+	ready 'Storage server modules initialised.'
+	directory './qabel-storage/'
+}
+
+task checkStorage {
+	File dataFolder = new File("./qabel-storage/data/");
+	if( !dataFolder.exists() ) {
+		dataFolder.mkdirs();
+	}
+	File nodeModulesFolder = new File("./qabel-storage/node_modules/");
+	if( !nodeModulesFolder.exists() ) {
+		getNodeModules.execute()
+	}
+}
+
+task startStorageServer(type: ExecWait) {
+	dependsOn checkStorage	
+	command 'node app.js &'
```

Comment:
```node``` is not the nodejs programm on my machine. I think you have to check for the nodejs installation and/or executable name first.